app/styles/_core.less

```diff
@@ -1029,6 +1029,12 @@ h1 small.meta, .build-config-summary .meta {
   color: @gray-light;
 }
 
+.no-background {
```

Comment:
A "global" fix that would address this any time a .table appears on a gray background would be:

.surface-shaded .table {
  background-color: transparent;
}

Plus, there is then no need to add an additional class to the table.